apps/src/code-studio/pd/application_dashboard/detail_view_contents.jsx

```diff
@@ -77,13 +80,19 @@ export default class DetailViewContents extends React.Component {
     });
   };
 
+  handleScoreChange = (event) => {
+    this.setState({
+      response_scores: Object.assign(this.state.response_scores, {[event.target.id]: event.target.value})
+    });
+  }
+
   handleSaveClick = () => {
     $.ajax({
       method: "PATCH",
       url: `/api/v1/pd/applications/${this.props.applicationId}`,
       dataType: 'json',
       contentType: 'application/json',
-      data: JSON.stringify(this.state)
+      data: JSON.stringify(Object.assign({}, this.state, {response_scores: JSON.stringify(this.state.response_scores)}))
```

Comment:
Why the nested `JSON.stringify` calls? Can't we just stringify the outer object?

Also, nit: prefer ES6 spread syntax `{...this.state}` over `Object.assign({}, this.state)`

Reply:
I'm converting it to a string because response_scores are stored as a string in the DB.